src/components/pages/public/MovieDetailPage.vue

```diff
@@ -58,6 +58,12 @@ export default {
     this.movieDetails = data;
     this.setMovieTitle(this.movieDetails.title);
   },
+  metaInfo() {
+    return {
+      title: `${this.movieDetails.title}`,
```

Comment:
what if there is no `movieDetails` yet? it will be set to `"  | montecinema"` - maybe it's worth to add some fallback?